app/assets/javascripts/sync.coffee.erb

```diff
@@ -128,7 +128,7 @@ class Sync.Pusher extends Sync.Adapter
     !!window.Pusher
 
   connect: ->
-    @client = new window.Pusher(Sync.PUSHER_API_KEY)
+    @client = <%= defined?(PusherFake) ? "new window.Pusher(#{[PusherFake.configuration.key, PusherFake.configuration.to_options].map(&:to_json).join(",")})" : "new window.Pusher(Sync.PUSHER_API_KEY)" %>
```

Comment:
@chrismccord, when I looked at the adapter configuration it didn't seem as if it was flexible enough to support `PusherFake`.  The resulting .js from above ends up looking like:

```
return this.client = new window.Pusher("fake_key_here", {
  "wsHost": "127.0.0.1",
  "wsPort": 50917,
  "enabledTransports": ["ws"]
});
```

The alternate route would then be expand out the existing config into something like:

    @client = new window.Pusher(Sync.PUSHER_API_KEY, {"wsHost": Sync.PUSHER_HOST, "wsPort": Sync.PUSHER_PORT, "enabledTransports": Sync.PUSHER_TRANSPORTS})

having those new config values default to the corresponding Pusher values.  Would you rather this PR do that?

@ajb, I can definitely split it up if this method is OK.